virtcontainers/cgroups.go

```diff
@@ -28,15 +28,15 @@ type sandboxCgroups struct {
 func (s *Sandbox) newCgroups() error {
 	// New will still succeed when cgroup exists
 	// create common parent for all kata-containers
-	// e.g. /sys/fs/cgroup/cpu/vc
+	// e.g. /sys/fs/cgroup/cpu/kata
```

Comment:
Even if it's not a big deal, this change and the one below should not be part of this commit.